docs/source/user-guide/installation.md

```diff
@@ -84,7 +84,7 @@ See the [Comet Kubernetes Guide](kubernetes.md) guide.
 
 Make sure `SPARK_HOME` points to the same Spark version as Comet was built for.
 
-```console
+```shell
```

Comment:
This fixes an issue where previously it was not possible to copy and paste the `$` in `$SPARK_HOME`